stubs/pluggy/pluggy/_result.pyi

```diff
@@ -0,0 +1,23 @@
+from collections.abc import Callable
+from types import TracebackType
+from typing import Generic, TypeVar
+from typing_extensions import TypeAlias
+
+_ExcInfo: TypeAlias = tuple[type[BaseException], BaseException, TracebackType]
```

Comment:
You can replace this with `from _typeshed import ExcInfo`:

https://github.com/python/typeshed/blob/032f9195f9e3788a6e5c7ecb086f173a3ac92a95/stdlib/_typeshed/__init__.pyi#L254